.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "recipes-core/lcd-tools/lcd-tools/third-party/cxxopts"]
```

Comment:
Have your OpenEmbedded recipe clone source code with SRC_URI = "git://...", never add code in the OE layer.